Install_to_firefox.md

```diff
@@ -1,23 +1,23 @@
-## Install to firefox from Mozilla Add-ons
+## Installing on Firefox from Mozilla Add-ons
 
-Go to [extension page](https://addons.mozilla.org/en-US/firefox/addon/wsse-generator/) 
+1. Go to the [extension page](https://addons.mozilla.org/en-US/firefox/addon/wsse-generator/) 
```

Comment:
I do not think is necessary enumerate this topic because there is just one step.